src/test/java/handler/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,38 @@
+package handler;
+
+import dao.StorageDao;
+import dao.StorageDaoImpl;
+import db.Storage;
+import model.FruitTransaction;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static Storage storage;
+    private static StorageDao storageDao;
+    private static OperationHandler operationHandler;
+    private static FruitTransaction fruitTransaction;
+
+    @BeforeClass
+    public static void beforeClass() {
+        storage = new Storage();
+        storageDao = new StorageDaoImpl(storage);
+        operationHandler = new BalanceOperationHandler(storageDao);
+    }
+
+    @Test
+    public void balanceOperation_ok() {
```

Comment:
```suggestion
    public void handle_ok() {
```
follow naming conventions